makerdao/governance-portal-v2
# Feature/sc 533/add mkr in chief to executive system info

### Link to Clubhouse story

https://app.shortcut.com/dux-makerdao/story/533/add-mkr-in-chief-to-executive-system-info

### What does this PR do?

Adds "MKR in chief" to the sidebar system info for executive pages

### Steps for testing:

1. Go to executive page
2. See new stat in sidebar ✨ 

### Screenshots (if relevant):
<img width="429" alt="Screen Shot 2021-11-04 at 2 27 59 PM" src="https://user-images.githubusercontent.com/5225766/140321583-f0659317-1697-426c-86fc-40ded143acaf.png">

### Any additional helpful information?:

Also refactored some hooks into the MKR module

### Add a GIF:
![](https://media.giphy.com/media/3ornjJRhOL0YwW7pG8/giphy.gif)
